CMakeLists.txt

```diff
@@ -208,6 +197,14 @@ else()
   message(STATUS "SIMD support disabled")
 endif()
 
+# Compiler options for different platforms and compilers
+if(MSVC)
+    target_compile_options(ipc_toolkit PRIVATE /W4 /MP)
+    target_compile_definitions(ipc_toolkit PRIVATE NOMINMAX)
```

Comment:
This is already defined at line 130:
```cmake
target_compile_definitions(ipc_toolkit PUBLIC NOMINMAX)
```
I like the change to make it only defined on MSVC and PRIVATE, so just delete lines 125–131.